lib/repo/mapping.go

```diff
@@ -10,15 +10,15 @@ import (
 
 const missingRepo = "Device repo is missing"
 
+var dm *deviceCollection = nil
```

Comment:
__[golint]__
* [golint] should drop = nil from declaration of var dm; it is the zero value

<!-- Comment by [SideCI](https://sideci.com) -->